test/infrastructure/api-stack.test.ts

```diff
@@ -0,0 +1,31 @@
+import * as cdk from 'aws-cdk-lib';
```

Comment:
I tried to run this tests but I am getting the following error, any idea why?:
![image](https://user-images.githubusercontent.com/46347630/148132371-308a2fe0-37b9-4c74-933f-e8cd51512474.png)
![image](https://user-images.githubusercontent.com/46347630/148132417-50f8cbd4-1818-4e67-8d73-05757d5f4ab8.png)

Reply:
Are these tests running against the real resources?

Reply:
This looks like a path issue. I'll try to dig further.